rules/opentitan.bzl

```
@@ -961,14 +1114,14 @@ def opentitan_flash_binary(
 
     This macro is mostly a wrapper around the `opentitan_binary` macro, but also
     creates artifacts for each of the keys in `PER_DEVICE_DEPS`, and if signing
-    is enabled, each of the keys in `signing_keys`. The actual artifacts created
+    is enabled, each of the keys in `signing_keysets`. The actual artifacts created
     artifacts created are outputs of the rules emitted by the `opentitan_binary`
     macro and those listed below.
     Args:
       @param name: The name of this rule.
       @param devices: List of devices to build the target for.
       @param platform: The target platform for the artifacts.
-      @param signing_keys: The signing keys for to sign each BIN file with.
+      @param signing_keysets: The signing keys for to sign each BIN file with.
```

nit: pre-existing typo
```suggestion
      @param signing_keysets: The signing keys to sign each BIN file with.
```

---

fixed, thanks!